Lib/test/test_pyrepl/test_pyrepl.py

```diff
@@ -1406,6 +1406,9 @@ class TestDumbTerminal(ReplTestCase):
     def test_dumb_terminal_exits_cleanly(self):
         env = os.environ.copy()
         env.pop('PYTHON_BASIC_REPL', None)
+        # Ignore PYTHONSTARTUP to not pollute the output
+        # with an unrelated traceback. See #137568
```

Comment:
It doesn't matter that much, but we like to use "GH-" to differentiate between BPO issues and a hypothetical future issue tracker if we ever move off GitHub.

```suggestion
        # with an unrelated traceback. See GH-137568.
```

Reply:
Thanks! Will remember that one :-)